common/lib/xmodule/xmodule/js/spec/video/video_progress_slider_spec.js

```diff
@@ -40,7 +40,7 @@
                 });
 
                 it('add ARIA attributes to time control', function () {
-                    var timeControl = $('div.slider > a');
```

Comment:
@muzaffaryousaf Correct. jQuery UI defaults to an `a` for the slider. I've got some markup here to ensure that we use a `div` instead.